source/modules/MEI.visual.xml

```diff
@@ -428,6 +428,15 @@
   <classSpec ident="att.mdiv.vis" module="MEI.visual" type="atts">
     <desc xml:lang="en">Visual domain attributes.</desc>
   </classSpec>
+  <classSpec ident="att.damp.vis" module="MEI.visual" type="atts">
+    <desc xml:lang="en">Visual domain attributes.</desc>
+    <classes>
+      <memberOf key="att.placementRelStaff"/>
+      <memberOf key="att.verticalGroup"/>
```

Comment:
```suggestion

      <memberOf key="att.verticalGroup"/>
```

Reply:
I don’t see the intention behind this suggestion… please help me @rettinghaus